zgw_consumers/migrations/0020_service_timeout.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 5.0.2 on 2024-02-21 08:29
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("zgw_consumers", "0019_alter_service_uuid"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="service",
+            name="timeout",
+            field=models.PositiveSmallIntegerField(
+                default=10,
+                help_text="Timeout used for the calls expressed in seconds",
+                validators=[django.core.validators.MaxValueValidator(60)],
```

Comment:
This max value seems arbitrary and I know systems exist where this is at times not enough. Should we even set an upper bound?